docs/DeveloperGuide.md

```diff
@@ -477,6 +500,15 @@ testers are expected to do more *exploratory* testing.
 
 1. _{ more test cases …​ }_
 
+### Adding an item
+1. Adding an item
```

Comment:
I think it'll be even better if you split up into 2 sections in this adding part: Adding to an existing item, adding to a non-existing item (instead of explaining them in the expected part)

Reply:
Done! That's a good idea thank you!